.github/workflows/develop-deploy.yml

```diff
@@ -0,0 +1,27 @@
+name: Develop Firebase functions deploy
```

Comment:
Initially I wanted to keep all the deploys in one workflow, but it proved not super straightforward to do 😕 (particularly choosing which service account secret/environment to use), so I've split them up